websub-ballerina/build.gradle

```diff
@@ -200,14 +200,15 @@ task ballerinaBuild {
                     commandLine 'sh', '-c', "$distributionBinPath/ballerina test --offline --all -- --b7a.config.file=src/websub_advanced_integration_tests/tests/advanced_services/sample-users.toml --test.hub.url=https://localhost:23191/websub/hub"
                 }
             }
-        }
-        exec {
-            workingDir project.projectDir
-            environment "JAVA_OPTS", "-DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true"
-            if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-                commandLine 'cmd', '/c', "$distributionBinPath/ballerina.bat build --offline --skip-tests --code-coverage -c ${moduleName}"
-            } else {
-                commandLine 'sh', '-c', "$distributionBinPath/ballerina build --offline --skip-tests --code-coverage -c ${moduleName}"
+        } else {
```

Comment:
Any reason to add an else clause here?

Reply:
if the project hasn't "skipBallerinaTests" property, the source is compiled two times. So I have added else clause

Reply:
No, that is fine during the dev time. This was a special case where we have to run tests in each module. The else part is not needed.